joomla/joomla-cms
# [3.7] Add a Postinstall Message to warn users on nearly unsupported envoirments

Pull Request for 3.7.x
### Summary of Changes

Based on the Joomla 4 Manifesto by @wilsonge https://developer.joomla.org/news/658-joomla4-manifesto.html i have created a postinstall warning to all not supported versions.
### Testing Instructions
- apply this patch
  Fire this SQL statment (replace #__ by your prefix)

``` sql
INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description_key`, `action_key`, `language_extension`, `language_client_id`, `type`, `action_file`, `action`, `condition_file`, `condition_method`, `version_introduced`, `enabled`)
VALUES
(700, 'COM_CPANEL_MSG_JOOMLA40_PRE_CHECKS_TITLE', 'COM_CPANEL_MSG_JOOMLA40_PRE_CHECKS_BODY', '', 'com_cpanel', 1, 'message', '', '', 'admin://components/com_admin/postinstall/joomla40checks.php', 'admin_postinstall_joomla40checks_condition', '3.7.0', 1);
```

If you have a unspported version or use mssql you get a postinstall like this:

![image](https://cloud.githubusercontent.com/assets/2596554/19008150/8e977446-8769-11e6-919d-602dbcaeb49d.png)

If you want you can test this but you have no unsupported version you can change e.g. the PHP Version check to  `version_compare(PHP_VERSION, '8.5.9', 'lt');`
### Documentation Changes Required

None
